mlflow/utils/autologging_utils/versioning.py

```diff
@@ -28,6 +28,12 @@ def _check_version_in_range(ver, min_ver, max_ver):
     return Version(min_ver) <= Version(ver) <= Version(max_ver)
 
 
+def _check_spark_version_in_range(ver, min_ver, max_ver):
```

Comment:
Can we add a short docstring to this function explaining what it does and why it's necessary?

Reply:
added along with an example to explain why we're doing this.